java/Oleksii-Riabov/src/com/zoolatech/lecture3/tasks/_2/NumberValidator.java

```diff
@@ -0,0 +1,28 @@
+package com.zoolatech.lecture3.tasks._2;
+
+public class NumberValidator extends AccountValidator {
```

Comment:
The task description contained a list of validation rules you needed to implement:

```
List of validation rules: string is not empty, valid email, valid phone number, number is in the range between X and Y, number is bigger than a value X,  number is less than a value X.
```

Reply:
Already change task to meet this requirements.